src/reducers/index.js

```diff
@@ -446,6 +450,31 @@ export default combineReducers({
     return state
   },
 
+  tagInvitationEditor: (state = {}, action) => {
```

Comment:
let's start adding new reducers to separate files -- reducers/index is too big already